skfda/_neighbors/base.py

```diff
@@ -508,14 +483,15 @@ def fit(self, X, y):
             self._sample_points = X.sample_points
             self._shape = X.data_matrix.shape[1:]
 
-            if not self.sklearn_metric:
+            if not self.multivariate_metric:
 
                 if self.metric == 'l2':
                     metric = lp_distance
                 else:
                     metric = self.metric
 
-                sk_metric = _to_sklearn_metric(metric, self._sample_points)
+                sk_metric = _to_multivariate_metric(metric,
```

Comment:
Maybe `sk_metric` should change its name too. With this change it could be confusing now.

Reply:
Some like ``sklearn_multivariate_metric``?  (``multivariate_metric`` it is used as an argument of the estimator)

Reply:
Maybe that is too long. But at least `sklearn_metric`, since the sk prefix is common for many scikits.

Reply:
Applied in c1fbcc3.